### PR TITLE
Remove unused notifications, their messages, and images

### PR DESCRIPTION
**Pull request by HowardBraham**
## **Description**

Continues the work that @legobeat was doing in #20703 and #21533

All old numbered notifications except 8, 20, and 24 were permanently turned off, but their code, their `messages.json`, and most importantly their images were still present in the Extension.

By removing all this, we reduce the size of the zipped Extension by 949kB, and the unzipped Extension by 1,852kB.

**IMPORTANT**: This is **NOT** well-tested from a QA perspective, and QA should do some good testing on it before we merge.

## **Related issues**

Followup to: #20703
Followup to: #21533

## **Pre-merge author checklist**

- [x] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/CODING_GUIDELINES.md).
- [x] I've clearly explained what problem this PR is solving and how it is solved.
- [x] I've linked related issues
- [ ] I've included manual testing steps
- [ ] I've included screenshots/recordings if applicable
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [x] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.
- [x] I’ve properly set the pull request status:
  - [ ] In case it's not yet "ready for review", I've set it to "draft".
  - [x] In case it's "ready for review", I've changed it from "draft" to "non-draft".

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

**Comment by sleepytanya** [comment]
Manual test cases passed:
- [x] Send native token MM
- [x] Send native token Dapp
- [ ]

**Comment by sleepytanya** [comment]
1. 'Watch NFT' and 'Watch all NFTs' is not responsive in Chrome, works as expected on Firefox.
2. ENS name resolution on Chrome and Firefox:
doesn't work on Linea Goerli and Sepolia
lineaGoerliEns
works on Goerli
goerliEns
On mainnet ENS name is initially recognized then 'ens lookup failed' message pops up


https://github.com/MetaMask/metamask-extension/assets/104780023/27166438-27ca-4a31-882d-3ad7251e2904

3. Errors in the background when cancelling a swap:
Firefox

https://github.com/MetaMask/metamask-extension/assets/104780023/5e05a727-798b-4002-af20-348fce93ee2e

Chrome
Screenshot 2024-02-02 at 8 34 44 PM
I wasn't able to cancel any swap (performed about 10 swaps).

**Review by davidmurdoch — DISMISSED** [review]

File: shared/notifications/index.js [hunk 1]
```diff
@@ -477,7 +148,6 @@ export const getTranslatedUINotifications = (
           )
         : '',
     },
-    ///: BEGIN:ONLY_INCLUDE_IF(blockaid)
```
[the remark this answers is not in the document]
Comment (davidmurdoch) [reply shared/notifications/index.js:480]: It got it working fine for webpack (er, well... source maps aren't correct yet).... I just really don't like it (although I admit the DX of conditionally including `import`s are better than any alternatives I've seen proposed).

[the remark this answers is not in the document]
Comment (HowardBraham) [reply shared/notifications/index.js]: I restored all of the blockaid codeguards.  We'll put it in a different PR.

**Review by legobeat — APPROVED** [review]

**Comment by chloeYue** [comment]
I did some exploratory testing for notifications and compared the notifications between the PR branch and the develop branch. QA passed. ✔️ @HowardBraham

**Review by chloeYue — APPROVED** [review]

**Pull request merged**